core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -557,7 +557,7 @@ ConditionalTabletMutator requireSame(TabletMetadata tabletMetadata, ColumnType t
      * second cause is ample resubmitting on unknown as mentioned above. Below are a few examples
      * that go over how Ample will handle these different situations.
      *
-     * <h3>Example 1</h3>
+     * <h4>Example 1</h4>
```

Comment:
I think it's better not to use headers in javadocs. Can just start a new paragraph and have the `<ul>` immediately follow. Javadoc is really strict about the use of headers because it uses them itself. I don't think we really need to use them most of the time.

Reply:
That's fair, but I was just trying to fix the build. I used the PR mechanism to test that all checks passed successfully. The commit to include the headings was done long ago, I think a recent change in the build caused it to fail. Update to JDK 17 maybe? If we want to remove all HTML headings from all javadoc, then it might be useful to create a separate ticket as I'm assuming we would want to add a check to prevent them from being reintroduced.

Reply:
I don't think we need to ban them. Your fix is fine. The build checks should be sufficient to catch the errors. I was just trying to extract a general principle from the situation. I think people can still use them if they need to.